### PR TITLE
Enable for suse and corrected for windows service

### PR DESCRIPTION
**Pull request by jcalonsoh**
### Description

Enable to install on suse 12+ and correct regex call on creating service on windows os

### Issues Resolved

Installing on windows server node 2012R2

### Check List

- [x] All tests pass. See <https://github.com/chef-cookbooks/community_cookbook_documentation/blob/master/TESTING.MD>
- [x] New functionality includes testing.
- [x] New functionality has been documented in the README if applicable
- [x] All commits have been signed for the Developer Certificate of Origin. See <https://github.com/chef-cookbooks/community_cookbook_documentation/blob/master/CONTRIBUTING.MD>

### CODE REVIEW

File: .delivery/build_cookbook/.kitchen.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
```
Comment (iennae) [inline .delivery/build_cookbook/.kitchen.yml]: Can you separate out the build_cookbook commits into a separate PR.

File: recipes/default.rb [hunk 2]
```diff
@@ -25,8 +25,8 @@
 end
 
 case node['platform_family']
-when 'windows', 'debian', 'rhel'
+when 'windows', 'debian', 'rhel', 'suse'
   include_recipe 'push-jobs::install'
 else
-  raise 'This cookbook currently supports only Windows, Debian-family Linux, and RHEL-family Linux.'
+  raise 'This cookbook currently supports only Windows, Debian-family Linux, RHEL-family Linux, ans Suse.'
```
Comment (iennae) [inline recipes/default.rb]: %s/ans/and/

File: .delivery/build_cookbook/.kitchen.yml (hunk 1 above)
Comment (jcalonsoh) [inline .delivery/build_cookbook/.kitchen.yml]: I remove it, and I proposed gitattribute merge ours startegy driver

File: recipes/default.rb (hunk 2 above)
Comment (jcalonsoh) [inline recipes/default.rb]: I will correct the typo

**Comment by jcalonsoh** [comment]
PR change to https://github.com/chef-cookbooks/push-jobs/pull/118
